From 7f7a8ab6b1ed2bdeef68efdbfd74b121f63c5d09 Mon Sep 17 00:00:00 2001
From: "Christopher N. Hesse" <raymanfx@gmail.com>
Date: Wed, 15 Mar 2017 15:13:35 +0100
Subject: [PATCH] zero: Unbreak camera permissions

 * Drop m2mscaler group, we don't have it.
 * Grant world read/write perms for video50,51:
   Cameraserver needs access to these nodes as well.

I have no idea how this works for Samsung.
Scaler units are used by both camera and media, so we
cannot assign it to either media or cameraserver owner
and have 0660 permissions.

Change-Id: Ibfccdffd03988de7113ca78766be4b55b6a90d3e
---
 rootdir/etc/ueventd.samsungexynos7420.rc | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/rootdir/etc/ueventd.samsungexynos7420.rc b/rootdir/etc/ueventd.samsungexynos7420.rc
index 319eb94..f257631 100644
--- a/rootdir/etc/ueventd.samsungexynos7420.rc
+++ b/rootdir/etc/ueventd.samsungexynos7420.rc
@@ -19,10 +19,10 @@
 /dev/video27              0660   media      graphics
 
 # scalers
-/dev/video50              0660   media      graphics
-/dev/video51              0660   media      graphics
-/dev/m2m1shot_scaler0     0660   cameraserver     m2mscaler
-/dev/m2m1shot_scaler1     0660   cameraserver       m2mscaler
+/dev/video50              0666   media      graphics
+/dev/video51              0666   media      graphics
+/dev/m2m1shot_scaler0     0660   cameraserver     graphics
+/dev/m2m1shot_scaler1     0660   cameraserver       graphics
 
 /dev/v4l-subdev*          0660   system     camera
 /dev/media0               0660   system     camera
